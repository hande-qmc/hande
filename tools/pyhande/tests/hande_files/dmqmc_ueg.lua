-- Create output with:
-- $[HANDE DIR]/bin/hande.x dmqmc_ueg.lua > dmqmc_ueg.out 2> dmqmc_ueg.err
-- Note that these settings are just for testing...
sys = ueg {
    dim = 3,
    nel = 14,
    ms = 0,
    cutoff = 1,
}

dmqmc {
    sys = sys,
    qmc = {
        tau = 0.05,
        rng_seed = 1472,
        init_pop = 200,
        mc_cycles = 2,
        nreports = 2,
        target_population = 100,
        state_size = 50000,
        spawned_state_size = 5000,
    },
    dmqmc = {
        beta_loops = 2,
    }
}
