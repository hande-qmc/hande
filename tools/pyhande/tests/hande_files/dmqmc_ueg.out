
   HANDE

 ================================================================
Version information
-------------------
Version       | v1.4-25-g4efb802
Commit hash   | 4efb80270c639ebaf202fe7b326e94a9ccfefafe
Commit author | Verena Neufeld
Commit date   | Thu Mar 21 12:08:44 2019 +0000
Branch        | master

Configuration and build information
-----------------------------------
Who compiled             | verena
Compilation hostname     | verena-Think
Operating system         | Linux-4.4.0-176-generic
CMake version            | 3.10.2
CMake generator          | Unix Makefiles
Build type               | release
Configuration time       | 2020-03-31 15:44:09 [UTC]
Python version           | 3.6.10
Fortran compiler         | /usr/bin/gfortran
C compiler               | /usr/bin/gcc
DET_SIZE set to          | 64
POP_SIZE set to          | 64
dSFMT Mersenne exponent  | 19937
MPI parallelization      | OFF
OpenMP parallelization   | OFF
Lua version              | 5.3.5
HDF5 library version     | 1.10.2

Further components
------------------
DISABLE_HDF5 not defined.  HDF5 enabled.
DISABLE_LANCZOS defined.  Lanczos disabled.
DISABLE_UUID not defined.  UUID enabled.
PARALLEL not defined.  MPI parallelization disabled.
DISABLE_SCALAPACK not defined.  ScaLAPACK enabled.
SINGLE_PRECISION not defined.  Double precision used throughout.
USE_POPCNT not defined. Internal POPCNT procedure used.

Runtime information
-------------------
Hostname:
    verena-Think
Current working dir:
    /home/verena/code3/hande/tools/pyhande/pyhande/tests/hande_files
Started running on 01/04/2020 at 21:55:24
 Calculation UUID: 8226e3f0-ee64-4128-9569-225da1f8b913.
 ================================================================

 Input options
 -------------

-- Create output with:
-- $[HANDE DIR]/bin/hande.x dmqmc_ueg.lua > dmqmc_ueg.out 2> dmqmc_ueg.err
-- Note that these settings are just for testing...
sys = ueg {
    dim = 3,
    nel = 14,
    ms = 0,
    cutoff = 1,
}

dmqmc {
    sys = sys,
    qmc = {
        tau = 0.05,
        rng_seed = 1472,
        init_pop = 200,
        mc_cycles = 2,
        nreports = 2,
        target_population = 100,
        state_size = 50000,
        spawned_state_size = 5000,
    },
    dmqmc = {
        beta_loops = 2,
    }
}

 -------------

 Basis functions
 ---------------

 Spin given in units of 1/2.
 k-points given in terms of the reciprocal lattice vectors of the crystal cell.

 index   k-point            ms       <i|h|i>
     1   (  0,  0,  0)      1        0.00000000E+00
     2   (  0,  0,  0)     -1        0.00000000E+00
     3   (  0,  0, -1)      1        1.30773168E+00
     4   (  0,  0, -1)     -1        1.30773168E+00
     5   (  0, -1,  0)      1        1.30773168E+00
     6   (  0, -1,  0)     -1        1.30773168E+00
     7   ( -1,  0,  0)      1        1.30773168E+00
     8   ( -1,  0,  0)     -1        1.30773168E+00
     9   (  1,  0,  0)      1        1.30773168E+00
    10   (  1,  0,  0)     -1        1.30773168E+00
    11   (  0,  1,  0)      1        1.30773168E+00
    12   (  0,  1,  0)     -1        1.30773168E+00
    13   (  0,  0,  1)      1        1.30773168E+00
    14   (  0,  0,  1)     -1        1.30773168E+00
    15   (  0, -1, -1)      1        2.61546336E+00
    16   (  0, -1, -1)     -1        2.61546336E+00
    17   ( -1,  0, -1)      1        2.61546336E+00
    18   ( -1,  0, -1)     -1        2.61546336E+00
    19   (  1,  0, -1)      1        2.61546336E+00
    20   (  1,  0, -1)     -1        2.61546336E+00
    21   (  0,  1, -1)      1        2.61546336E+00
    22   (  0,  1, -1)     -1        2.61546336E+00
    23   ( -1, -1,  0)      1        2.61546336E+00
    24   ( -1, -1,  0)     -1        2.61546336E+00
    25   (  1, -1,  0)      1        2.61546336E+00
    26   (  1, -1,  0)     -1        2.61546336E+00
    27   ( -1,  1,  0)      1        2.61546336E+00
    28   ( -1,  1,  0)     -1        2.61546336E+00
    29   (  1,  1,  0)      1        2.61546336E+00
    30   (  1,  1,  0)     -1        2.61546336E+00
    31   (  0, -1,  1)      1        2.61546336E+00
    32   (  0, -1,  1)     -1        2.61546336E+00
    33   ( -1,  0,  1)      1        2.61546336E+00
    34   ( -1,  0,  1)     -1        2.61546336E+00
    35   (  1,  0,  1)      1        2.61546336E+00
    36   (  1,  0,  1)     -1        2.61546336E+00
    37   (  0,  1,  1)      1        2.61546336E+00
    38   (  0,  1,  1)     -1        2.61546336E+00

 Number of electrons: 14
 Number of basis functions: 38

 Bit-length of integers used to store determinant bit-strings: 64
 Number of integers used to store determinant bit-strings: 1


 Reference determinant and so symmetry sector selected using the Aufbau principle.
 Selected symmetry  1.

 DMQMC
 -----

 WARNING: renormalised excitation generators not implemented.
 WARNING: If this upsets you, please send patches.

 Memory allocated per core for main walker list (MB):      1.80
 Number of elements per core in main walker list: 50000
 Memory allocated per core for spawned walker lists (MB):      0.24
 Number of elements per core in spawned walker list: 5000

 # Starting the excitation generator initialisation.
 # Finishing the excitation generator initialisation, time taken:  0.0000000000E+00
 -- Start JSON block --
 {
     "system": {
         "nbasis": 38,   
         "nel": 14,   
         "nvirt": 24,   
         "Ms": 0,   
         "nalpha": 7,   
         "nbeta": 7,   
         "nvirt_alpha": 12,   
         "nvirt_beta": 12,   
         "nsym": 19,   
         "sym0": 1,   
         "sym_max": 19,   
         "nsym_tot": 19,   
         "sym0_tot": 1,   
         "sym_max_tot": 19,   
         "symmetry": 1,   
         "tot_sym": false,   
         "aufbau_sym": true,   
         "max_number_excitations": 14,   
         "ueg": {
             "r_s": 1.00000000,   
             "ecutoff": 1.00000000,   
             "k_fermi": 1.91915829,   
             "E_fermi": 1.84158428,   
             "ktwist": [0.00000000,0.00000000,0.00000000],
             "L": [3.88512994,3.88512994,3.88512994]
         }
     },
     "qmc": {
         "rng_seed": 1472,   
         "real_amplitudes": false,   
         "real_amplitude_force_32": false,   
         "spawn_cutoff": 0.01000000,   
         "excit_gen":  "renorm",   
         "pattempt_update": false,   
         "pattempt_zero_accum_data": false,   
         "pattempt_single": 0.00000000,   
         "pattempt_double": 1.00000000,   
         "pattempt_parallel": 0.00000000,   
         "tau": 0.05000000,   
         "tau_search": false,   
         "vary_shift_from": 0.00000000,   
         "vary_shift_from_proje": false,   
         "initial_shift": 0.00000000,   
         "shift_damping": 0.05000000,   
         "walker_length": 50000,   
         "spawned_walker_length": 5000,   
         "D0_population": 200.00000000,   
         "target_particles": 100.00000000,   
         "target_reference": false,   
         "initiator_approx": false,   
         "initiator_pop": 3.00000000,   
         "ncycles": 2,   
         "nreport": 2,   
         "power_pitzer_min_weight": 0.01000000,   
         "quasi_newton": false,   
         "quasi_newton_threshold": 0.00001000,   
         "quasi_newton_value": 1.00000000,   
         "use_mpi_barriers": false    
     },
     "dmqmc": {
         "beta_loops": 2,   
         "replica_tricks": false,   
         "start_av_rdm": 0,   
         "weighted_sampling": false,   
         "vary_weights": false,   
         "find_weights": false,   
         "find_weights_start": 0,   
         "calc_excit_dist": false,   
         "all_sym_sectors": false,   
         "all_spin_sectors": false,   
         "initiator_level": -1,   
         "sampling_probs":  "[]",   
         "finish_varying_weights": 0,   
         "fermi_temperature": false,   
         "target_beta": 1.00000000,   
         "mom_dist_kmax": 0.00000000,   
         "struc_fac_qmax": 0.00000000    
     },
     "ipdmqmc": {
         "ipdmqmc": false,   
         "initial_matrix":  "hartree_fock",   
         "grand_canonical_initialisation": false,   
         "symmetric": true,   
         "chem_pot": 0.00000000,   
         "metropolis_attempts": 0    
     },
     "rdm": {
         "nrdms": 0,   
         "spawned_length": 0,   
         "doing_rdm": false,   
         "calc_ground_rdm": false,   
         "calc_inst_rdm": false,   
         "doing_concurrence": false,   
         "doing_vn_entropy": false,   
         "output_rdm": false    
     },
     "operators": {
         "energy": false,   
         "energy_squared": false,   
         "kinetic_energy": false,   
         "potential_energy": false,   
         "H0_energy": false,   
         "HI_energy": false,   
         "correlation_fn": false,   
         "staggered_mad_ind": false,   
         "rdm_r2": false,   
         "full_r2": false,   
         "mom_dist": false    
     },
     "restart": {
         "read_restart": false,   
         "read_id": 2147483647,   
         "write_restart": false,   
         "write_id": 2147483647,   
         "write_freq": 2147483647,   
         "write_restart_shift": false,   
         "write_shift_id": 2147483647,   
         "restart_rng": true    
     },
     "load balancing": {
         "nslots": 1,   
         "pop": 1000,   
         "percent": 0.05000000,   
         "max_attempts": 2,   
         "write_info": false    
     },
     "reference": {
         "det": [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14],
         "det_ms": 0,   
         "det_symmetry": 1,   
         "H00": 13.60355734,   
         "F0": 15.69278015,   
         "hilbert_space_det": [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14],
         "hilbert_space_det_ms": 0,   
         "hilbert_space_det_symmetry": 1,   
         "ex_level": 14    
     }
 }
 -- End JSON block --

 Information printed out every QMC report loop:

 Shift: the energy offset calculated at the end of the report loop.
 Trace: The current total population on the diagonal elements of the density matrix.
 # particles: current total population of Hamiltonian particles.
 # states: number of many-particle states occupied.
 # spawn_events: number of successful spawning events across all processors.
 R_spawn: average rate of spawning across all processors.
 time: average time per Monte Carlo cycle.

 Note that all particle populations are averaged over the report loop.

 #     iterations   Instant shift         Trace                 # particles                # states  # spawn_events   R_spawn    time    
                0   3.4159842485E-01      0.0000000000E+00      2.0000000000E+02                100              48    0.2495    0.0060  
                2   6.1038996132E-01      0.0000000000E+00      1.0100000000E+02                 58              33    0.2598    0.0000  
                4   7.8032387413E-01      0.0000000000E+00      5.9000000000E+01                 41              26    0.2743    0.0000  
 # Resetting beta... Beta loop = 2
 # Resetting random number generator with a seed of: 1473.
 # WARNING: more than 3 particles spawned in a single event 2 times in the last report loop.
 # Mean number of particles created in blooms:      3.0
 # This warning only prints  1 time(s). You may wish to reduce the time step.
                0   3.4159842485E-01      0.0000000000E+00      2.0000000000E+02                 97              45    0.2515    0.0060  
                2   3.7751129214E-01      0.0000000000E+00      1.0100000000E+02                 93              46    0.2414    0.0000  
                4   4.6443475704E-01      0.0000000000E+00      9.4000000000E+01                 79              36    0.2506    0.0000  

 Timing breakdown
 ----------------

 Time for each calculation section (seconds):

 UEG initialisation: .00
 DMQMC calculation : .03

 ================================================================
 Finished running on 01/04/2020 at 21:55:24
 Wall time (seconds):                           0.03
 CPU time (per processor, seconds):             0.03
 ================================================================

